releasenotes/notes/bump-python-to-2715-7fc6de90afd1f365.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
```

Comment:
not clear to me why this should be in the upgrade notes instead of in the `enhancements`. This change should introduce no breaking change on users of the Agent.

Making the comment because I think the `upgrade` section should always only list "actionable items" for users.